src/app/components/blogList.tsx

```diff
@@ -0,0 +1,38 @@
+// src/components/ListItem.tsx
+'use client'
+
+import React from 'react';
+import { FaRegPlayCircle } from 'react-icons/fa';
```

Comment:
replace for hero icons

Reply:
I couldn't find this social media icons on hero Icons, that's why i got from font awesome